libs/react-helpers/src/lib/constants/links.ts

```diff
@@ -0,0 +1,32 @@
+/** ExternalLinks returns external documentation links that are specific to an environment
+ * @param {string} docsUrl - the documentation URL for the environment ex. https://docs.vega.xyz/testnet or https://docs.vega.xyz (NX_VEGA_DOCS_URL)
+ */
+
+export const DocsLinks = (docsUrl = 'https://docs.vega.xyz/testnet') => ({
```

Comment:
chore (blocking): 2 things here:

A) I think it would be better to not hard code this default parameter and instead explicitly require an argument to be passed in. This value normally comes from an .env config. If the link changes we now have two places to update making this harder to maintain

B) `DocsLink` is neither a class nor constructor function nor compoment, can it just be lowercase and perhaps more obviously a function. Perhaps `createDosLinks`?

Reply:
updated in last commit.